gensim/models/phrases.py

```diff
@@ -197,8 +221,10 @@ def add_vocab(self, sentences):
         # directly, but gives the new sentences a fighting chance to collect
         # sufficient counts, before being pruned out by the (large) accummulated
         # counts collected in previous learn_vocab runs.
-        min_reduce, vocab = self.learn_vocab(sentences, self.max_vocab_size, self.delimiter, self.progress_per)
+        min_reduce, vocab, total_words = \
+        self.learn_vocab(sentences, self.max_vocab_size, self.delimiter, self.progress_per)
```

Comment:
There's no hard limit; if the line becomes hard to read, we break it.

If the break would be even harder to read than the original, we don't break it.

Line continuations are indented at one extra level (4 spaces to the right).